README.md

```diff
@@ -61,90 +66,208 @@ SENTRY_DNS=8.8.8.8
-5. All API endpoints are rendered from `/api/`. Example `http://localhost:8080/api/admin` - Admin endpoint
+
+12. Starting the client
+
+    ```
+    make client-start
+    ```
+
+13. Client eslint checks
+
+    ```
+    make client-lint-check
+    ```
+
+14. Client eslint autofix
+
+    ```
+    make client-lint-fix
+    ```
+
+15. Client tests
+
+    ```
+    make client-test
+    ```
+
+16. All client sided routes are rendered from root, `/`. Example `http://localhost:5173/admin` - Admin React Page
+17. All API endpoints are rendered from `/api/`. Example `http://localhost:3000/api/admin` - Admin endpoint
 
 **NB: All requests are listening from port `8080`.**
 
 ## Technologies Used
+
 This application has been built with these technologies:
-* Ruby `v3.0.2`
-* Ruby on Rails `v7.0.4`
-* RSpec `v3.1.2`
-* React `v18.0`
-* Typescript `v4.9.5`
-* Jest `v29.5.0`
-* Firebase `v9.17.0`
-* nginx
-* Docker
-* GitHub Actions
-* GitHub Projects
 
+- Ruby `v3.0.2`
+- Ruby on Rails `v7.0.4`
+- RSpec `v3.1.2`
+- React `v18.0`
+- Typescript `v4.9.5`
+- Jest `v29.5.0`
+- Firebase `v9.17.0`
+- nginx
+- Docker
+- GitHub Actions
+- GitHub Projects
 
 ## Authors
-* [Ian Okumu](https://github.com/otsembo)
-* [Add creds here]
+
+- [Ian Okumu](https://github.com/otsembo)
+- [Naomi Rono](https://github.com/naomirono)
+- [Kimutai Kiprotich](https://github.com/kimutai01)
+- [Michael Munavu](https://github.com/MICHAELMUNAVU83)
+- [Awino Mackrine](https://github.com/mackrineawino)
+- [Timothy Kipngetich Kiyeng](https://github.com/timothykiyeng)
+- [Enid](https://github.com/MissDine)
+- [Cynthia Chelangat](https://github.com/chelahcynthia)
+- [Tosh](https://github.com/Muriithi-Gitonga)
+- [Emmanuel Karingi](https://github.com/emmanuelkaringi)
+
 ## Contribution Guide
 
 In order to contribute to this repository, you need to follow these steps:
 
+- Create a new branch with your name and the feature you are working on. Example: `janedoe/reset-password`
 
-* Create a new branch with your name and the feature you are working on. Example: `janedoe/reset-password`
 ```git
 git branch name/feature
 ```
 
-* Checkout to your branch in order to work on a feature.
+- Checkout to your branch in order to work on a feature.
+
 ```git
 git checkout name/feature
 ```
 
-* Once you have completed your work on the feature, create a commit with a comprehensive message. Example:
+- Once you have completed your work on the feature, create a commit with a comprehensive message. Example:
+
 ```git
 git commit -m "Login: added JWT token in header"
 ```
 
-* Push the changes to the remote repository
+- Push the changes to the remote repository
+
 ```git
 git push -u origin name/feature
 ```
 
-* Ensure you create three environment files: `.env`, `.env.production` and `.env.development`.
+- Ensure you create three environment files: `.env`, `.env.production` and `.env.development`.
```

Comment:
Remove the `.env.development` and replace with `.env.test` as discussed in an earlier comment in this review.